libs/components/flyout/src/lib/modules/flyout/flyout.service.spec.ts

```diff
@@ -54,10 +59,7 @@ describe('Flyout service', () => {
     });
     service.open(SkyFlyoutHostsTestComponent);
     applicationRef.tick();
```

Comment:
Yes. Because the service doesn't have a component fixture that we can detect changes from - without these calls change detection on the components that the service opens does not fire. I did remove these calls from the component test file.